apps/web/components/ui/UsernameAvailability/index.tsx

```diff
@@ -51,7 +65,7 @@ export const UsernameAvailabilityField = ({
             setInputUsernameValue={onChange}
             onSuccessMutation={onSuccessMutation}
             onErrorMutation={onErrorMutation}
-            user={user}
+            addOnLeading={usernamePrefix}
```

Comment:
Refactored. Noticed we pass the full user just to determine the `addOnLeading`. Let's just pass the `addOnLeading` from here instead.